packages/database/src/DatabaseModel.js

```diff
@@ -78,7 +78,9 @@ export class DatabaseModel {
     if (!this.fieldNames) {
       const schema = await this.fetchSchema();
 
-      this.fieldNames = Object.keys(schema);
+      const customColumnSelectors = this.customColumnSelectors || {};
+
+      this.fieldNames = [...Object.keys(schema), ...Object.keys(customColumnSelectors)];
```

Comment:
This is so that the column selectors can easily be queried, so we can directly query on the db model when there are custom column selectors, and not just via the central server